appcenter-data/Data.js

```diff
@@ -19,13 +19,33 @@ const DefaultPartitions = {
 const Data = {
```

Comment:
You can add Jest tests as part of this PR.

Reply:
Added unit tests